core/src/main/java/hudson/PluginWrapper.java

```diff
@@ -226,6 +232,14 @@ public void setDependants(@Nonnull Set<String> dependants) {
         }
     }
 
+    /**
+     * Get the list of components that depend optionally on this plugin.
+     * @return The list of components that depend optionally on this plugin.
```

Comment:
nit: Not really useful to duplicate such JavaDoc, I would like to have only the `@return The list...` and the "global" JavaDoc does not add any information. (same for other methods)